app/models/user.rb

```diff
@@ -7,5 +7,9 @@ class User < ApplicationRecord
 
   has_many :reviews
 
-  validates_presence_of :first_name, :last_name
+  validates_presence_of :first_name, :last_name, :role
+
+  def admin?
```

Comment:
A question mark in a method implies that it will return a boolean, not a string.  If this method is for assigning the role, lose the ?